Marlin/src/lcd/e3v2/proui/dwin_defines.h

```diff
@@ -78,3 +78,29 @@
 #define DEF_BEDPIDT TERN(PREHEAT_1_TEMP_BED, PREHEAT_1_TEMP_HOTEND, 60)
 #define DEF_PIDCYCLES 5
 
+/**
+ * ProUI internal feature flags
+ */
+#if ALL(SDCARD_SORT_ALPHA, SDSORT_GCODE)
+  #define PROUI_MEDIASORT
+#endif
+#if ENABLED(POWER_LOSS_RECOVERY)
+  #define PROUI_ITEM_PLR    // Tune > Power-loss Recovery
+#endif
+#if ENABLED(HAS_JUNCTION_DEVIATION)
+  #define PROUI_ITEM_JD     // Tune > Junction Deviation
+#endif
+#if ENABLED(LIN_ADVANCE)
+  #define PROUI_ITEM_ADVK   // Tune > Linear Advance
+#endif
+#if HAS_MESH
+  #define PROUI_MESH_EDIT   // Add a menu to edit mesh points
+#endif
+#if ANY(PROUI_PID_TUNE, MPC_AUTOTUNE) && DISABLED(DISABLE_TUNING_GRAPH)
```

Comment:
unless `#include "../../../inc/Conditionals_post.h"` is included, `PROUI_PID_TUNE` isn't seen as defined, and so `PROUI_TUNING_GRAPH` is not enabled

Reply:
Right. Any header that refers to settings needs to include `MarlinConfigPre.h` so that is the solution I just applied.